README.md

```diff
@@ -1,187 +1,130 @@
-
-```php
-__()
-_n()
-_x()
-```
-
-```html
-<p>
-  <%= __('Click %shere%s!') | format('<a href="#">', '</a>') | raw %>
-</p>
-```
-
-```html
-<p>
-  <%= _n('deleted %d message', 'deleted %d messages', count) | format(count) %>
-  <!-- count === 1 -> "deleted 1 message" -->
-  <!-- count > 1 -> "deleted $count messages" -->
-</p>
-```
-
-- Handlebars.
-
-You can use Twig i18n functions in Handlebars, just load your template from a Twig view.
-
-# Build
-
-To build a plugin , run `./build.sh`.
-
-Some build process steps are described below (their dependencies etc.).
-
-## packtranslations step
-
-This step imports translations from Transifex and generates MO files. It requires:
-* `tx` client: https://docs.transifex.com/client/installing-the-client
-* `msgfmt` command (from Gettext package)
-Finally , a `WP_TRANSIFEX_API_TOKEN` environment variable should be initialized with a valid key.
-
-# Publish
-
-The `publish` command currently does the following:
-* Pushes translations POT file to Transifex;
-* Publishes the release in SVN.
-
-Before you run it, you need to:
-1. Ensure there is an up-to-date local copy of MailPoet SVN repository in `.mp_svn` directory by running `./do svn:checkout`.
-2. Have all your features merged in Git `master`, your `mailpoet.php` and `readme.txt` tagged with a new version.
-3. Run `./build.sh` to produce a `mailpoet.zip` distributable archive.
-
-Everything's ready? Then run `./do publish`.
-If the job goes fine, you'll get a message like this:
-```
-Go to '.mp_svn' and run 'svn ci -m "Release 3.0.0-beta.9"' to publish the
-release
-
-Run 'svn copy ...' to tag the release
-```
-It's quite literal: you can review the changes to be pushed and if you're satisfied, run the suggested command to finish the release publishing process.
-
-If you're confident, execute `./do publish --force` and your release will be published to the remote SVN repository without manual intervention (automatically). For easier authentication you might want to set `WP_SVN_USERNAME` and `WP_SVN_PASSWORD` environment variables.
-
-# Acceptance testing
+# Frameworks and libraries
+
+- [Paris ORM](https://github.com/j4mie/paris).
+- [Robo](https://robo.li/) is used to write and run workflow commands.
+- [Codeception](https://codeception.com/) is used to write unit and acceptance tests.
+- [Docker](https://www.docker.com/), [Docker Compose](https://docs.docker.com/compose/) and [Selenium](https://www.seleniumhq.org/) to run acceptance tests.
+- [React](https://reactjs.org/) is used to create most of UIs.
+- [Marionette](https://marionettejs.com/) is used to build the newsletters editor.
+- [Stylus](http://stylus-lang.com/) is used to write styles.
+- [Mocha](https://mochajs.org/), [Chai](https://www.chaijs.com/) and [Sinon](https://sinonjs.org/) are used to write Javascript tests.
+- [ESLint](https://eslint.org/) is used to lint JS files.
+- [Webpack](https://webpack.js.org/) is used to bundle assets.
+
+# Files structure
```

Comment:
Why do you think we need this section here? It will get outdated and I don't see the benefits.

Reply:
My idea was to show where each type of source code goes (PHP, JS and CSS). I think this global structure does not change that much to be outdated.

Reply:
I understand, but everyone can find where each type of source goes. If you cannot find where PHP lives you probably shouldn't contribute. And I can see a couple of files that changed quite recently. Even the PHP version wasn't up-to-date so we cannot count on ourselves.